The logos widget in the Starterkit (#2393)
umbraco/UmbracoDocs: Add-ons/Starter-Kit-TEMP/Widgets/index.md

Below you can see how a ***Contact Form*** will look per default:

![movePage.jpg](images/Contact-form-example.png)

### The Logos Widget

> The Logos Widget does what the name implies and lets you set up logos, the settings for your logos images.

Reviewer: Not sure about the last bit of this article - I've made the following suggestion instead:

```suggestion
The Logos Widget does what the name implies and lets you add a set of logos to your website.
```

> You can choose what images you will use for logos and what pages they need to link to And/or you can add text to it.

Reviewer: ```suggestion
You can choose what images you will use for logos and what pages they need to link to and/or you can add text to it.
```

> This widget is divided into two parts. Part one is for the heading and other general settings for the container. Part two is for adding logo images and settings for these.

Reviewer: ```suggestion
The configuration for this widget is divided into two parts:
```

> You will have these settings in the first section:

Reviewer: ```suggestion
The first part is for the heading and the container:
```

- Pre Heading
- Heading
- Text
- Container Width
- Compact
- Black & White
- Add content ( you need to do this to get the second section)
- Background Color
- Text Color

> You will have these settings in the second section:

Reviewer: ```suggestion
Part two is for adding logo and settings for these. For each logo you want to add, you have the following options:
```
